src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -899,6 +899,46 @@
     </Environment>
   </Provider>
 
+  <!--
+                                                    ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
+                                                    ██ ████ ▄▄▀██ ▄▄▀██ █▀▄██
+                                                    ██ ████ ▀▀ ██ ▀▀▄██ ▄▀███
+                                                    ██ ▀▀ █ ██ ██ ██ ██ ██ ██
+                                                    ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
+  -->
+
+  <Provider Name="Lark" DisplayName="Lark (Feishu)" Id="0822ff2a-c4e7-4e1d-a797-bb24632b3f1a"
```

Comment:
No strong opinion, but maybe `Lark/Feishu` would be better than `Lark (Feishu)`?

Reply:
That's fine 👍🏻